packages/@aws-cdk/aws-cloudfront/lib/web_distribution.ts

```diff
@@ -176,13 +242,15 @@ export interface SourceConfiguration {
    * The relative path to the origin root to use for sources.
    *
    * @default /
+   * @deprecated Use originPath on s3OriginSource or customOriginSource
```

Comment:
Not sure I understand this change. Can you explain why we shouldn't keep these properties in `SourceConfiguration` only?

Reply:
Origin and fallback Origin could have different path/header configurations which makes it necessary to move the setting to the origin and not on the `SourceConfiguration` level.